clients/src/main/java/org/apache/kafka/clients/consumer/MockConsumer.java

```diff
@@ -14,26 +14,29 @@
 
 import org.apache.kafka.clients.consumer.internals.NoOpConsumerRebalanceListener;
 import org.apache.kafka.clients.consumer.internals.SubscriptionState;
-import org.apache.kafka.common.Metric;
-import org.apache.kafka.common.MetricName;
-import org.apache.kafka.common.PartitionInfo;
-import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.*;
```

Comment:
I think your IDE did this automatically and we normally stick to the no wildcard rule?